code/game/objects/items/crayons.dm

```diff
@@ -151,6 +150,28 @@
 	colour = "#DA00FF"
 	colourName = "purple"
 
+/obj/item/toy/crayon/red/syndicate
+
+/obj/item/toy/crayon/red/syndicate/attack(mob/M, mob/user)
```

Comment:
This is copied code. Best to change the existing code to take in a taste string or such.